k8s-operator/sessionrecording/hijacker.go

```diff
@@ -157,52 +184,6 @@ func (h *spdyHijacker) setUpRecording(ctx context.Context, conn net.Conn) (net.C
 	return lc, nil
 }
 
-// CastHeader is the asciicast header to be sent to the recorder at the start of
```

Comment:
`CastHeader` is now in ./sessionrecording and is shared for by ssh/tailssh and kube session recording